pkg/server/server.go

```diff
@@ -159,7 +163,23 @@ func NewServer(cfg Config, stopper *stop.Stopper) (*Server, error) {
 		// Use a non-annotated context here since the annotation just looks funny,
 		// particularly to new users (made worse by it always printing as [n?]).
 		log.Shout(context.Background(), log.Severity_WARNING,
-			"running in insecure mode, this is strongly discouraged. See --insecure.")
+			"RUNNING IN INSECURE MODE!")
+		host, port, _ := net.SplitHostPort(s.cfg.Addr)
+		if host == "" {
+			host = "<your IP address>"
+		}
+		addr := net.JoinHostPort(host, port)
+		log.Shout(context.Background(), log.Severity_WARNING,
+			"'INSECURE' IS REALLY INSECURE:\n"+
+				"- your cluster is OPEN for any client that can access "+addr+"!\n"+
+				"- any user even 'root' can log in WITHOUT PROVIDING A PASSWORD!\n"+
```

Comment:
any user, even 'root', can log in...

(To my ear, this isn't correct without the commas.)